src/containers/eventRSVP/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { Alert, Space, Spin, Table, Typography } from 'antd';
+import React, { useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import { connect, useDispatch } from 'react-redux';
+import { Link as RouterLink, useParams } from 'react-router-dom';
+import { ChungusContentContainer } from '../../components';
+import { C4CState } from '../../store';
+import {
+  asyncRequestIsComplete,
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+} from '../../utils/asyncRequest';
+import { getUpcomingEvents } from '../upcoming-events/ducks/thunks';
+import { EventsReducerState } from '../upcoming-events/ducks/types';
+import { getEventRegistrations } from './ducks/thunks';
+import { EventRegistrationsReducerState, Registration } from './ducks/types';
+
+const { Title, Link } = Typography;
+interface EventRSVPProps {
```

Comment:
Hmm I feel like prettier won't like this spacing.

Reply:
![image](https://user-images.githubusercontent.com/23691775/113761642-b13a5780-96e5-11eb-900e-48d72abb1091.png)
‼️ my extension made it look like there was a space